softwareconfiguration/mlplan/core/src/main/java/ai/libs/mlplan/core/PipelineEvaluator.java

```diff
@@ -104,18 +106,18 @@ public Double evaluateSupervised(final IComponentInstance c) throws InterruptedE
 
 			ITimeTrackingLearner trackableLearner = new TimeTrackingLearnerWrapper(c, learner);
 			if (c instanceof ComponentInstance) {
-				trackableLearner.setPredictedInductionTime(((ComponentInstance)c).getAnnotation(IEvaluationSafeGuard.ANNOTATION_PREDICTED_INDUCTION_TIME));
-				trackableLearner.setPredictedInferenceTime(((ComponentInstance)c).getAnnotation(IEvaluationSafeGuard.ANNOTATION_PREDICTED_INFERENCE_TIME));
+				trackableLearner.setPredictedInductionTime(((ComponentInstance) c).getAnnotation(IEvaluationSafeGuard.ANNOTATION_PREDICTED_INDUCTION_TIME));
+				trackableLearner.setPredictedInferenceTime(((ComponentInstance) c).getAnnotation(IEvaluationSafeGuard.ANNOTATION_PREDICTED_INFERENCE_TIME));
 			}
 
 			if (this.logger.isDebugEnabled()) {
-				this.logger.debug("Starting benchmark {} for classifier {}", this.benchmark, (learner instanceof ScikitLearnWrapper) ? learner.toString() : learner.getClass().getName());
+				this.logger.debug("Starting benchmark {} for classifier {}", this.benchmark, (learner instanceof IScikitLearnWrapper) ? learner.toString() : learner.getClass().getName());

```

Comment:
This is ugly.The pipeline evaluator should not know about any specific implementation at this point.

Reply:
what do you suggest?